usr/share/rear/conf/examples/SLE11-SLE12-SAP-HANA-UEFI-example.conf

```diff
@@ -1,48 +1,52 @@
-#
-# Install EFI ISO creation tool:
-# For UEFI the ebiso package is required to use ReaR on SLES 11 and SLES 12.
-# The ebiso package is available at
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/
-# For example to download the ebiso package for SLES 12 use a command like (the RPM package version and release numbers change):
-#   wget http://download.opensuse.org/repositories/Archiving:/Backup:/Rear/SLE_12/x86_64/ebiso-0.1.4-1.2.x86_64.rpm
-# and install it with a command like:
-#   rpm -ivh ebiso-0.1.4-7.1.x86_64.rpm
-# Furthermore ebiso needs ReaR higher than 1.17.2 e.g. use a development snapshot release from
-# http://download.opensuse.org/repositories/Archiving:/Backup:/Rear:/Snapshot/
+
 # Create the ReaR recovery system as ISO image:
 OUTPUT=ISO
-# Use ebiso to make an UEFI bootable ISO image of the ReaR recovery system:
-ISO_MKISOFS_BIN=/usr/bin/ebiso
-# Store the backup file via NFS on a NFS server:
+
+# Use /usr/bin/xorrisofs (provided by the RPM package xorriso)
+# to create a UEFI bootable ISO image of the ReaR recovery system:
+ISO_MKISOFS_BIN="/usr/bin/xorrisofs"
+
+# To boot the ReaR recovery system with UEFI Secure Boot
+# specify a (properly signed) first stage bootloader for Secure Boot:
+#SECURE_BOOT_BOOTLOADER="/boot/efi/EFI/sles/shim.efi"
+
+# Store the backup tar archive via NFS on a NFS server:
 BACKUP=NETFS
-# BACKUP_OPTIONS variable contains the NFS mount options and
-# with 'mount -o nolock' no rpc.statd (plus rpcbind) are needed:
+
+# BACKUP_OPTIONS specifies the NFS mount options.
+# With 'mount -o nolock' no rpc.statd (plus rpcbind) are needed:
 BACKUP_OPTIONS="nfsvers=3,nolock"
+
 # If the NFS server is not an IP address but a hostname,
-# DNS must work in the ReaR recovery system when the backup is restored.
+# DNS must work in the ReaR recovery system:
 BACKUP_URL=nfs://your.NFS.server.IP/path/to/your/rear/backup
+
 # Keep an older copy of the backup in a HOSTNAME.old directory
-# provided there is no '.lockfile' in the HOSTNAME directory:
+# (provided there is no '.lockfile' in the HOSTNAME directory):
 NETFS_KEEP_OLD_BACKUP_COPY=yes
-# Make sure to exclude all SAP HANA related volume groups and mount points - modify next line with your HANA mount points:
+
+# Exclude all SAP HANA related volume groups and mount points - adapt it to your specific HANA mount points:
 EXCLUDE_VG=( vgHANA-data-HC2 vgHANA-data-HC3 vgHANA-log-HC2 vgHANA-log-HC3 vgHANA-shared-HC2 vgHANA-shared-HC3 )
 BACKUP_PROG_EXCLUDE+=( '/hana/*' )
+
+# Let the ReaR recovery system run dhclient to get an IP address
+# instead of using the same IP address as your original system:
+#USE_DHCLIENT="yes"
+
 # Set a root password for the ReaR recovery system in a confidential way
 # to allow SSH connection without a public/private key pair.
-# Never use your original root password here:
+# Never use your original system's root password here:
 #{ SSH_ROOT_PASSWORD='password_on_the_rear_recovery_system' ; } 2>>/dev/$SECRET_OUTPUT_DEV
```

Comment:
As above, I'd leave this out if not required for HANA specifically.

Reply:
Done via
https://github.com/rear/rear/pull/3306/commits/bb4b850d54ee82dcf5aa9503259b8bafc3b8c578